pytransform3d/transform_manager/_transform_graph_base.pyi

```diff
@@ -0,0 +1,66 @@
+import abc
+from typing import Dict, Tuple, List, Hashable, Any
+import scipy.sparse as sp
+import numpy as np
+import numpy.typing as npt
+
+
+class TransformGraphBase(abc.ABC):
+    strict_check: bool
+    check: bool
+    nodes: List[Hashable]
+    i: List[int]
+    j: List[int]
+    transform_to_ij_index = Dict[Tuple[Hashable, Hashable], int]
+    connections: sp.csr_matrix
+    dist: np.ndarray
+    predecessors: np.ndarray
+    _cached_shortest_paths: Dict[Tuple[int, int], List[Hashable]]
+
+    def __init__(self, strict_check: bool = ...,
+                 check: bool = ...) -> "TransformGraphBase": ...
+
+    @property
+    @abc.abstractmethod
+    def transforms(self) -> Dict[Tuple[Hashable, Hashable], np.ndarray]: ...
+
+    @abc.abstractmethod
+    def _check_transform(self, A2B: Any) -> Any: ...
+
+    @abc.abstractmethod
+    def _invert_transform(self, A2B: Any) -> Any: ...
```

Comment:
If the `self.transforms` property always returns numpy-array, do we need `_invert_transform` method? We can call the utility function directly in `get_transform()` method. This would be OK with Temporal Manager too